packages/client/vite.config.ts

```diff
@@ -87,11 +87,10 @@ function targetBuildPlugin(dev: boolean, target: string): Plugin {
         const candidatePath = path.join(
           // If imported file starts with /src we can not append it to importer dir
           // so we need to strip the path by the rootPrefix first
-          importedFile.dir.startsWith("/src") &&
-            importerFile.dir.includes(rootPrefix)
-            ? `${importerFile.dir.split(rootPrefix)[0]}/`
+          importedFile.dir.startsWith(sourceRootPrefix) &&
```

Comment:
this can go away, ofc, but it's instructive for now, to see how the candidate is derived